src/assembler.m

```diff
@@ -128,6 +136,27 @@
     else
       error('wrong hyperElasModel for frame element.')
 		end
+    
+    if aeroBool
+    userWindVel = "windVel";
```

Comment:
sugiero revisar la indentación acá y en las 3 líneas superiores

Reply:
This hardcode version must change

Reply:
ok, supongo que no está resuelto por ahora...

Reply:
Done